.gitignore

```diff
@@ -34,7 +34,7 @@ hfuzz_workspace
 **/.DS_Store
 .cache
 js/packages/web/.env
-js/packages/candy-machine-ui/.env
+js/packages/candy-machine-ui/.env.example
```

Comment:
it needs to be the reverse. The .example should be added but the normal .env should be ignored

Reply:
yea my bad didn't notice